CHANGES

```diff
@@ -14,6 +14,7 @@
 - API Addition: Desktop Sound now returns number of channels and sample rate.
 - API Addition: PointSpriteParticleBatch blending is now configurable.
 - TOOLS Features: Blending mode and sort mode can be changed in Flame particle 3D editor.
+- API Addition: ObjLoader now supports ambientColor, ambientTexture, transparencyTexture, specularTexture and shininessTexture
```

Comment:
well, 1.10.0 is already out :-) so you can start a new 1.10.1 chapter at the beginning of the file